### PR TITLE
Create energy context

### PR DESCRIPTION
**Pull request by gasserluc**

### CODE REVIEW

**Review by junjou — APPROVED** [review]

File: energy/jsonld-contexts/energy-compound.jsonld [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "@context": [
+        "https://easy-global-market.github.io/ngsild-api-data-models/device/jsonld-contexts/device.jsonld",
+        "https://easy-global-market.github.io/ngsild-api-data-models/energy/jsonld-contexts/energy.jsonld",
+        "https://easy-global-market.github.io/ngsild-api-data-models/environment/jsonld-contexts/environment.jsonld",
+        "https://easy-global-market.github.io/ngsild-api-data-models/alertEventRisk/jsonld-contexts/alertEventRisk.jsonld",
+        "https://easy-global-market.github.io/ngsild-api-data-models/authorization/jsonld-contexts/authorization.jsonld",
+        "https://easy-global-market.github.io/ngsild-api-data-models/mapping/jsonld-contexts/flowMapping.jsonld",
+        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context-v1.7.jsonld"
```
Comment (bobeal) [inline energy/jsonld-contexts/energy-compound.jsonld:9]: we are 1.8 now (things like `JsonProperty` won't work with 1.7)

File: energy/jsonld-contexts/energy.jsonld [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "@context": {
+        "Building": "https://vocab.egm.io/Building",
+        "Client": "https://vocab.egm.io/Client",
+        "Contract": "https://vocab.egm.io/Contract",
+        "DeliveryPoint": "https://vocab.egm.io/DeliveryPoint",
+        "Invoice": "https://vocab.egm.io/Invoice",
+        "Site": "https://vocab.egm.io/Site",
+        "Supplier": "https://vocab.egm.io/Supplier",
+        "amount": "https://vocab.egm.io/amount",
+        "city": "https://vocab.egm.io/city",
```
Comment (bobeal) [inline energy/jsonld-contexts/energy.jsonld:11]: for general things like `city`, `postalCode`, ... have a look at schema.org

Comment (gasserluc) [reply energy/jsonld-contexts/energy.jsonld:11]: Found for postalCode but not for city surprisingly. Has been modified

Comment (bobeal) [reply energy/jsonld-contexts/energy.jsonld:11]: there is https://schema.org/City (but it's more like an entity than a property)

**Review by bobeal — APPROVED** [review]

**Pull request merged**